src/main/java/core/basesyntax/util/FileReaderImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.util;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class FileReaderImpl implements FileReader {
+
+    @Override
+    public List<String> read(String filePath) {
+        try {
+            return Files.lines(Path.of(filePath))
+                    .collect(Collectors.toList());
+        } catch (IOException e) {
+            throw new RuntimeException("Cannot read file by path " + filePath);
```

Comment:
Consider including the original exception `e` in the `RuntimeException` to provide more context about the error. This can be done by using `throw new RuntimeException("Cannot read file by path " + filePath, e);`.